src/test/jsx/preprocess_examples.test.ts

```diff
@@ -139,6 +198,68 @@ describe("Preprocessing (examples)", () => {
             builder.element("span", { "data-foo": "'<&" }, builder.text(`"'<&`))
         );
 
+        check(
```

Comment:
@fnd Could you take a look if I missed some spread attribute test cases here? Something weird that I didn't think of?

Reply:
Seems sensible to me; I can't think of anything else either right now.